### PR TITLE
Fixing MathML expressions

### PR DESCRIPTION
**Pull request by jClugstor**
Fixed some MathML expressions in SEIRHD and SEIRHDS stockflow models

### CODE REVIEW

File: data/models/SEIRHDS_stockflow.json
Comment (sabinala) [inline data/models/SEIRHDS_stockflow.json:1]: In line 58, "lost" should be the parameter "los" (it stands for "length of stay").

File: data/models/SEIRHD_stockflow.json
Comment (sabinala) [inline data/models/SEIRHD_stockflow.json:1]: Same here, in line 58, "lost" should be "los."

**Review by sabinala — CHANGES_REQUESTED** [review]
Please change "lost" to "los" in line 58 of each file, per my comments.

**Comment by jClugstor** [comment]
Changes made, thanks for catching that

**Pull request merged**